nipoppy_cli/nipoppy/utils.py

```diff
@@ -167,18 +177,18 @@ def get_pipeline_tag(
     pipeline_name: str,
     pipeline_version: str,
     pipeline_step: Optional[str] = None,
-    participant: Optional[str] = None,
-    session: Optional[str] = None,
+    participant_id: Optional[str] = None,
+    session_id: Optional[str] = None,
     sep="-",
 ):
     """Generate a tag for a pipeline."""
```

Comment:
Tag?

Reply:
Like a string that fully identifies the pipeline run (name + version + step + participant + session). Couldn't think of a better name